src/runner/configuration.rs

```diff
@@ -0,0 +1,16 @@
+
+pub struct Configuration {
+    pub parallel: bool,
+}
+
+impl Configuration {
+    pub fn parallel(self, parallel: bool) -> Self {
+        Configuration { parallel: parallel }
```

Comment:
`Configuration { parallel : parallel, .. self }`, if possible, is more future-proof